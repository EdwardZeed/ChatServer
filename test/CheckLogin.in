LOGIN test1 ABC
REGISTER test1 ABC
q