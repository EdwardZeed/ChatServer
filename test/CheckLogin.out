RESULT LOGIN 0

RESULT REGISTER 1

